spec/lib/api/v3/cost_types/cost_type_representer_spec.rb

```diff
@@ -104,21 +104,21 @@
   context 'generation' do
     subject(:generated) { representer.to_json }
 
-    it { should include_json('CostType'.to_json).at_path('_type') }
+    it { is_expected.to include_json('CostType'.to_json).at_path('_type') }
 
     describe 'cost_type' do
-      it { should have_json_path('id') }
+      it { is_expected.to have_json_path('id') }
 
-      it { should have_json_path('name') }
+      it { is_expected.to have_json_path('name') }
 
-      it { should have_json_path('units') }
-      it { should have_json_path('unit') }
-      it { should have_json_path('unitPlural') }
+      it { is_expected.to have_json_path('units') }
+      it { is_expected.to have_json_path('unit') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.